lmdeploy/pytorch/kernels/cuda/fill_kv_cache.py

```diff
@@ -79,7 +47,7 @@ def _fill_kv_cache_kernel(
     QSeqLens,
     KVSeqLens,
     BlockOffsets,
-    num_heads: tl.constexpr,
+    is_decoding: tl.constexpr,
```

Comment:
Does it mean that the quant kernel of `fill_kv_cache` can also be optimized?

Reply:
quant kernel would be optimized in future.